akka-docs/rst/java/cluster-sharding.rst

```diff
@@ -0,0 +1,267 @@
+.. _cluster_sharding_java:
+
+Cluster Sharding
+================
+
+Cluster sharding is useful when you need to distribute actors across several nodes in the cluster and want to
+be able to interact with them using their logical identifier, but without having to care about
+their physical location in the cluster, which might also change over time.
```

Comment:
slightly tempted to cross link to persistence here - "often implemented using PersistentActors" or something like that